### DISCUSSION TITLE
Schedules not working properly

### DISCUSSION THREAD

**Discussion by flyboyusa** [tagyoureit/nodejs-poolController#954] [App and O/S Config Questions]
I have a schedule to run Pool twice per day, but the second schedule from 6pm-12pm doesn't fire.
All the others seem to work.
I've deleted this schedule and re-added it, but still the same problem.
Am I doing something wrong?

<img width="811" alt="Screenshot 2024-05-29 at 6 49 53 PM" src="https://github.com/tagyoureit/nodejs-poolController/assets/95569334/1dfb533b-e599-41d3-9281-fce13ff7ca00">

**Comment by rstrouse** [comment]
Do you mean to run the first schedule to run until midnight?

**Reply by flyboyusa** [reply]
Yes I wanted it 6pm to midnight.  I think you're right, the 12pm is noon, which is overlapping with the other schedule.
I've changed it to 12am and we'll see if it fires properly tonight.
Thanks!

**Reply by tagyoureit** [reply]
That should work.  You can also try to set it for 11:59pm so it doesn't overlap with the next day.

**Reply by flyboyusa** [reply]
I'm looking for some additional troubleshooting...my 6am-12noon schedule never starts on time at 6am...it's always somewhere between 6:10 and 6:15 am.  Any idea as to why it wouldn't fire at 6am on the nose?  The 6pm - midnight seems to be starting/stopping perfectly.

**Reply by tagyoureit** [reply]
We would need a replay capture to see the config and logs.
